example/main.go

```diff
@@ -27,7 +27,7 @@ import (
 
 	"mosn.io/proxy-wasm-go-host/proxywasm/common"
 	proxywasm "mosn.io/proxy-wasm-go-host/proxywasm/v1"
-	"mosn.io/proxy-wasm-go-host/wasmer"
+	"mosn.io/proxy-wasm-go-host/wazero"
```

Comment:
No problem.. it will work for now.

build tags can work as long as it is only wasmer. if ever wasmer and wasmtime they will need to be in different go.mod because they conflict in C https://github.com/wapc/wapc-go/pull/30

Reply:
ok made wasmer tag and ran build and test with it in CI (though currently no tests for it)